src/main/java/io/reactivex/Maybe.java

```diff
@@ -1974,7 +1974,12 @@ public final T blockingGet(T defaultValue) {
      */
     @SchedulerSupport(SchedulerSupport.NONE)
     public final <R> Maybe<R> compose(MaybeTransformer<T, R> transformer) {
-        return wrap(to(transformer));
+        try {
+            return wrap(transformer.apply(this));
```

Comment:
There is no need to try-catch it if it doesn't declare `throws Exception`. So either each transformer should specify `apply(...) throws Exception` or these are not needed.